bin/gen_viewer_json.py

```diff
@@ -1,12 +1,13 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
```

Comment:
I don't like this. You should run in a virtualenv.

Reply:
Fixed.